src/components/transactions/post-conditions/list.tsx

```diff
@@ -28,7 +28,10 @@ export const PostConditions: React.FC = () => {
   });
 
   const getPostConditionsContent = () => {
-    if (pendingTransaction?.txType === TransactionTypes.STXTransfer) {
+    if (
+      pendingTransaction?.txType === TransactionTypes.STXTransfer &&
+      postConditions.length === 0
+    ) {
```

Comment:
You say in the PR description that this was a "bug" - curious what made you think it's a bug? STX transfers cannot have post conditions - [per the SIP](https://github.com/stacksgov/sips/blob/main/sips/sip-005/sip-005-blocks-and-transactions.md#type-0-transferring-an-asset).

Reply:
ah yeah, I was testing this feature by adding post conditions to a stx-transfer, but maybe in a future PR we should alert to when that has been the case, and remove any added post-conditions